packages/swingset-runner/package.json

```diff
@@ -20,8 +20,10 @@
   },
   "dependencies": {
     "@agoric/assert": "^0.6.0",
+    "@agoric/deploy-script-support": "^0.10.3",
     "@agoric/ertp": "^0.16.2",
     "@agoric/internal": "^0.3.2",
+    "@agoric/inter-protocol": "^0.16.1",
```

Comment:
I'm a bit weirded out by having swingset-runner sit *above* inter-protocol/deploy-script-support in the dependency graph. In general, swingset code is unaware of chains or the Agoric economic layers, and if we intend to move all swingset-related packages to their own repo some day, this will be a sticking point.. we may need to leave swingset-runner behind in `agoric-sdk`.

We've talked about having a separate package which provides the benchmarking support: when we do that, please consider moving `swingset-runner/src/chain.js` and the `demo/vaultPerfTest/` stuff that provokes this dependency to that other package, and returning swingset-runner to only depend upon swingset stuff. Or maybe making a `@agoric/swingset-runner-chain` package to hold it.

Reply:
I share Brian's concern about contaminating swingset-runner with stuff that's not part of SwingSet (mainly `chain.js` -- I don't think `demo/vaultPerfTest` is as much of a concern as it's isolated), but given how things are structured right now there wasn't really any other graceful way to do it.  The longer term approach, I think, is for swingset-runner to support some kind of plugin scheme, so that the chain-specific stuff (or other hypothetical future uses for SwingSet) that needs to be loaded outside the SwingSet can be accommodated with code that would live in a different package, be tested independently, etc.  But that would involve playing various games with dynamic import and so on, and that's just not on the critical path at the moment.  I think what's in this PR is the best we're going to do for now.  However, I will create a separate issue to put in the backlog so we don't lose track.

Reply:
Update: added #8278 to address this.